src/lib/irap_export_ascii.cpp

```diff
@@ -0,0 +1,68 @@
+#include "include/irap_export.h"
+#include <cmath>
+#include <fstream>
+#include <iomanip>
+#include <ostream>
+#include <sstream>
+
+void write_header_ascii(const irap_header& header, std::ostream& out) {
+  out << std::setprecision(6) << std::fixed << std::showpoint;
+  out << "-996 " << header.ny << " " << header.xinc << " " << header.yinc
+      << "\n";
+  out << header.xori << " " << header.xmax << " " << header.yori << " "
+      << header.ymax << "\n";
+  out << header.nx << " " << header.rot << " " << header.xrot << " "
+      << header.yrot << "\n";
+  out << "0  0  0  0  0  0  0\n";
+}
+
+void write_values_ascii(surf_span values, std::ostream& out) {
+  out << std::setprecision(4) << std::fixed << std::showpoint;
+  size_t written_on_line = 0;
+  auto rows = values.extent(0);
+  auto cols = values.extent(1);
+  for (size_t j = 0; j < cols; j++) {
+    for (size_t i = 0; i < rows; i++) {
+      auto v = values(i, j);
+      if (std::isnan(v))
```

Comment:
I guess this can be inlined:
```
out << (isnan(v) ? UNDEF_MAP_IRAP : v);
```

Reply:
made both of the ifs inside the loop into ternary ifs